vmdb/app/models/registry_item.rb

```diff
@@ -14,7 +14,7 @@ def self.add_elements(miq_set, scan_item, vm, xmlNode)
     return if hashes.nil?
 
     new_reg = []
-    deletes = vm.registry_items.find(:all, :select=>"id, name").collect {|rec| [rec.id, rec.name]}
+    deletes = vm.registry_items.pluck(:id, :name)
```

Comment:
I think it will re-query. But I'm not sure whether there's a good reason for that.